test_scripts/API/VehicleData/common.lua

```diff
@@ -46,14 +43,21 @@ do
   m.json = actions.json
 end
 
---[[ Common Variables ]]
+--[[ Common Constants and Variables ]]
 m.rpc = {
   get = "GetVehicleData",
   sub = "SubscribeVehicleData",
   unsub = "UnsubscribeVehicleData",
   on = "OnVehicleData"
 }
 
+m.rpcHMIMap = {
+  [m.rpc.get] = "VehicleInfo.GetVehicleData",
+  [m.rpc.sub] = "VehicleInfo.SubscribeVehicleData",
+  [m.rpc.unsub] = "VehicleInfo.UnsubscribeVehicleData",
+  [m.rpc.on] = "VehicleInfo.OnVehicleData"
+}
+
 m.vd = {
```

Comment:
Is it possible to run tests for a specific vehicle data items? From looks of it, these changes generalize the VD testing for all params and there is no way to run tests for a singular or a set of VD items.

Reply:
Currently this can be achieved by commenting values in `m.vd` table.

Reply:
Ok, but that is not really clean approach. Is it possible to set some kind of environment variable or configuration for that affect. As a test runner, i'd not like to modify the lua scripts to target specific test runs.

Reply:
That's a good idea. I'm going to push an update regarding this in a few hours.